### PR TITLE
ROX-16884 Add alerts when cpu/memory commitment per availability zone…

### PR DESCRIPTION
**Pull request by ludydoo**
… exceeds thresholds

### CODE REVIEW

File: resources/mixins/kubernetes/jsonnetfile.lock.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
           "subdir": "grafana-builder"
         }
       },
-      "version": "1cb605505efe5a630f00993ff611a241fdcc264f",
+      "version": "d303b2031264728728dd1e1c05f74f67027139f6",
```
Comment (stehessel) [inline resources/mixins/kubernetes/jsonnetfile.lock.json:21]: Why did you update the mixin?

File: resources/prometheus/prometheus-rules.yaml [hunk 2]
```diff
@@ -476,3 +476,141 @@ spec:
             rhacs_org_id: "{{ $labels.rhacs_org_id }}"
             rhacs_cluster_name: "{{ $labels.rhacs_cluster_name }}"
             rhacs_environment: "{{ $labels.rhacs_environment }}"
+    - name: az-resources
+      rules:
+        - alert: WorkerNodesByAZMemoryQuotaOverCommit
+          expr: |
+            (
```
Comment (stehessel) [inline resources/prometheus/prometheus-rules.yaml:483]: I think these promQL queries a bit too lengthy. Could you try to break them down with recording rules?

**Comment by stehessel** [comment]
Can you add unit tests for the new alerts?

**Comment by stehessel** [comment]
Also I'd recommend to set up your editor to clip unnecessary whitespaces ;-)

File: resources/prometheus/prometheus-rules.yaml (hunk 2 above)
Comment (stehessel) [reply resources/prometheus/prometheus-rules.yaml:483]: Maybe also keep in mind that we'd like to use some of these queries as part of our dashboard, so the shorter the better 😅

File: resources/mixins/kubernetes/jsonnetfile.lock.json (hunk 1 above)
Comment (ludydoo) [reply resources/mixins/kubernetes/jsonnetfile.lock.json:21]: I just ran `make` and it did that all by itself, perhaps a new version?

**Review by stehessel — APPROVED** [review]
LGTM 🚀

**Pull request merged**